nvchecker/util.py

```diff
@@ -100,6 +100,10 @@ def __init__(
 
   def get_key(self, name: str, legacy_name: Optional[str] = None) -> Optional[str]:
     '''Get the named key (token) in the keyfile.'''
+    if self.keys.get(legacy_name):
+      logger.warning(
+        "You're using a legacy key name for your token, please check the docs to updade."
+        " Legacy keys will be removed, eventually.", key=legacy_name)
```

Comment:
I have no intention to change the keyfile names.

Reply:
Ok, no problem. I was presuming that having a two-prone api in the code would be something that you'd like to eventually expulse from it.

I'll remove the warning, then. Generally, I think using the `host` as key is the most specific way of declaring them (for future keys).

Reply:
Prefixing with the `source` name makes it easy to see where it is going to be used.